dowhy/interpreters/confounder_distribution_intepreter.py

```diff
@@ -0,0 +1,58 @@
+import seaborn as sns
```

Comment:
Is it possible to work without the seaborn package? In the propensity_balance_interpreter, I used this line to have the seaborn asthetics without having to import it. 
plt.style.use("seaborn-white")

If that works, that will be great, to keep a minimal list of packages as requirements for dowhy.

Reply:
yes definitely :) I will change it matplotlib